griptape/memory/tool/storage/blob_artifact_storage.py

```diff
@@ -0,0 +1,31 @@
+from attr import define, field
+from griptape.artifacts import BaseArtifact, ListArtifact, BlobArtifact, InfoArtifact
+from griptape.memory.tool.storage import BaseArtifactStorage
+
+
+@define
+class BlobArtifactStorage(BaseArtifactStorage):
```

Comment:
Should this be named `LocalBlobArtifactStorage`? Or will the storage destinations come in the form of Drivers in the future? i.e. `LocalBlobArtifactStorageDriver`, `S3BlobArtifactStorageDriver`

Reply:
Yeah, I'd like to keep storages high level and then add driver support for individual storage backends later.